dev/tools/gen_keycodes/data/keyboard_map_gtk_cc.tmpl

```diff
@@ -0,0 +1,24 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <map>
+
+// DO NOT EDIT -- DO NOT EDIT -- DO NOT EDIT
+// This file is generated by flutter/flutter@dev/tools/gen_keycodes/bin/gen_keycodes.dart and
+// should not be edited directly.
+//
+// Edit the template dev/tools/gen_keycodes/data/keyboard_maps_gtk_cxx.tmpl instead.
```

Comment:
Is this the right path to edit?

Reply:
Yep, it's wrong. Turns out it wasn't the only case though - https://github.com/flutter/flutter/pull/60015.